app/models/replaceable.rb

```diff
@@ -14,7 +14,7 @@ def create_or_replace(payload, &block)
       end
 
       retry_strategy.call do
-        item.assign_attributes(payload.except(:id, :version))
+        item.assign_attributes(payload.except(:id, :version, :access_limited))
```

Comment:
We should get rid of version here as that column no longer exists on content item models.